### ISSUE TITLE
Files missing in v4 release

### ISSUE THREAD

**Issue by jbeezley** [tunnckoCore/parse-function#29]
After installing the latest release, I get the following
```javascript
> require('parse-function')
Error: Cannot find module './lib/utils'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (node_modules/parse-function/index.js:14:15)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
```

Inspecting `node-modules/parse-function`, the `lib` directory is not present.
```
$ ls node_modules/parse-function/
CHANGELOG.md  index.js  LICENSE  node_modules  package.json  README.md
```

**Comment by tunnckoCore** [comment]
damn, really, missed that. good catch, ill update

**Comment by jbeezley** [comment]
Thanks for the quick fix!

**Comment by tunnckoCore** [comment]
Publish will be delayed for a bit, until #30 :)

Thanks for the usage and report!

**Comment by tunnckoCore** [comment]
Published as v4.0.1 and will publish 4.0.2 for #31
